.vib/discourse/goss/goss.yaml

```diff
@@ -0,0 +1,30 @@
+file:
+  /opt/bitnami/discourse/tmp:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: discourse
+    group: root
+  /opt/bitnami/discourse/plugins:
+    filetype: symlink
+    linked-to: /bitnami/discourse/plugins
```

Comment:
What does this mean? I'm just curious if it's something I could use in other tests.